webpack/components/extensions/HostDetails/Tabs/ErrataTab/ErrataTab.js

```diff
@@ -135,6 +135,14 @@ export const ErrataTab = () => {
     initialSearchQuery: searchParam || '',
   });
 
+  const tdSelect = useCallback((errataId, rowIndex) => ({
+    disable: !isSelectable(errataId),
+    isSelected: isSelected(errataId),
+    onSelect: (event, selected) => selectOne(selected, errataId),
+    rowIndex,
+    variant: 'checkbox',
+  }), [isSelectable, isSelected, selectOne]);
+
```

Comment:
We likely need a useCallback like this in all places that are using the useSelectionSet hook no? or does your useCallback on the isSelected method handling that issue by itself. 
Would this change then be needed? or is this for exhaustive deps only?

Reply:
Tbh I don't think the callback does much here- I can test it and see what the improvement is. I kept it mainly because of your preference for putting that object in a function rather than having an object literal down in the JSX..

Reply:
I was fully ready to delete this and revert back to the object in JSX, but then I tested it..

Tested with `tdSelect useCallback`: 174ms 178ms 171ms, average 174.33ms
With object literal in JSX: 202ms 185ms 200ms, average 195.67ms

Adds 21ms on average.

Also I feel like I can _feel_ the difference a bit as well!  For that reason I'm for keeping the useCallback. thoughts?

(Fyi, PF's doc for it is here: https://www.patternfly.org/v4/components/table/#tdselecttype)